src/Symfony/Component/Translation/DataCollector/TranslationDataCollector.php

```diff
@@ -60,11 +61,11 @@ public function reset()
     }
 
     /**
-     * @return array
+     * @return Data
      */
     public function getMessages()
     {
-        return isset($this->data['messages']) ? $this->data['messages'] : [];
+        return isset($this->data['messages']) ? $this->data['messages'] : new Data([]);
```

Comment:
this might change an `if (message)` in the twig template, can you please confirm this doesn't break anything?
otherwise I'm good with array|Data - other collector do it I think

Reply:
```diff
--- a/src/Symfony/Bundle/WebProfilerBundle/Resources/views/Collector/translation.html.twig
+++ b/src/Symfony/Bundle/WebProfilerBundle/Resources/views/Collector/translation.html.twig
@@ -71,7 +71,7 @@
 
     <h2>Messages</h2>
 
-    {% if collector.messages is empty %}
+    {% if not collector.messages|length %}
         <div class="empty">
             <p>No translations have been called.</p>
         </div>
```